config/filter.d/roundcube-auth.conf

```diff
@@ -15,7 +15,7 @@ before = common.conf
 
 prefregex = ^\s*(\[\])?(%(__hostname)s\s*(?:roundcube(?:\[(\d*)\])?:)?\s*(<[\w]+>)? IMAP Error)?: <F-CONTENT>.+</F-CONTENT>$
 
-failregex = ^(?:FAILED login|Login failed) for <F-USER>.*</F-USER> from <HOST>(?:(?:\([^\)]*\))?\. (?:(?! from ).)*(?: user=(?P=user))? in \S+\.php on line \d+ \(\S+ \S+\))?$
+failregex = ^(?:FAILED login|Login failed) for <F-USER>.*</F-USER></F-USER>(?:against \S+)? from <HOST>(?:(?:\([^\)]*\))?\. (?:(?! from ).)*(?: user=(?P=user))? in \S+\.php on line \d+ \(\S+ \S+\))?$
```

Comment:
This format will be already matched by current RE (probably due to `(?:(?:\([^\)]*\))?\. (?:(?! from ).)*`)...
But OK, I'll merge it after small review because of new tests and for the consistence. Thx.

By the way, there is accidentally additional `</F-USER>`, what would bother the RE (since unmatched tag causing also unmatched closing parenthesis)...

Reply:
Fixed now in 8e0a2366f067ee33d0840b57fcd2810b3281b378, tests passed too, so I'll merge it. Thx again.